usvm-python/src/test/kotlin/org/usvm/samples/DictsTest.kt

```diff
@@ -0,0 +1,228 @@
+package org.usvm.samples
+
+import org.junit.jupiter.api.Test
+import org.usvm.UMachineOptions
+import org.usvm.machine.types.PythonAnyType
+import org.usvm.runner.PythonTestRunnerForPrimitiveProgram
+import org.usvm.test.util.checkers.eq
+import org.usvm.test.util.checkers.ignoreNumberOfAnalysisResults
+
+class DictsTest : PythonTestRunnerForPrimitiveProgram("Dicts", UMachineOptions(stepLimit = 60U)) {
+    @Test
+    fun testExpectDict() {
+        check1WithConcreteRun(
+            constructFunction("expect_dict", listOf(PythonAnyType)),
+            eq(2),
+            standardConcolicAndConcreteChecks,
+            /* invariants = */ emptyList(),
+            /* propertiesToDiscover = */ listOf(

```

Comment:
Why don't you use named arguments?

Reply:
Because these functions are functional properties, they have no argument names.

Reply:
OK, but in future, prefer to replace functional properties with member functions. Yes, it may cause boilerplate code, but it improves readability and you won't have to manually add comments with argument names.